packages/core/src/domain/configuration.ts

```diff
@@ -215,18 +249,28 @@ function getEndpoint(type: string, conf: TransportConfiguration, source?: string
   return `https://${host}/v1/input/${conf.clientToken}?${parameters}`
 }
 
-export function isIntakeRequest(url: string, configuration: Configuration) {
-  return (
-    getPathName(url).indexOf('/v1/input/') !== -1 &&
-    (haveSameOrigin(url, configuration.logsEndpoint) ||
-      haveSameOrigin(url, configuration.rumEndpoint) ||
-      haveSameOrigin(url, configuration.traceEndpoint) ||
-      (!!configuration.internalMonitoringEndpoint && haveSameOrigin(url, configuration.internalMonitoringEndpoint)) ||
-      (!!configuration.replica &&
-        (haveSameOrigin(url, configuration.replica.logsEndpoint) ||
-          haveSameOrigin(url, configuration.replica.rumEndpoint) ||
-          haveSameOrigin(url, configuration.replica.internalMonitoringEndpoint))))
-  )
+function getHost(intakeType: IntakeType, endpointType: EndpointType, site: string) {
+  const middleDomain = INTAKE_MIDDLE_DOMAINS[intakeType]
+  const endpoint = ENDPOINTS[intakeType][endpointType]
+  return `${endpoint}${middleDomain}${site}`
+}
```

Comment:
Could we make it support host for new datacenter as well?
cf us3 use case

Reply:
I think all new datacenters hosts follow the pattern `${endpoint}.browser-intake-${domain}` so as long as the `userConfiguration.site` equals the `domain`, it should work. Am I missing something?

Reply:
yes, us3 case